src/modules/Login/action.js

```diff
@@ -179,9 +181,13 @@ export const initializeAccount = (account: EdgeAccount, touchIdInfo: Object) =>
 
     if (accountInitObject.customTokens) {
       accountInitObject.customTokens.forEach(token => {
-        accountInitObject.customTokensSettings.push(token)
-        // this second dispatch will be redundant if we set 'denomination' property upon customToken creation
-        accountInitObject.denominationKeys.push({ currencyCode: token.currencyCode, denominationKey: token.multiplier })
+        if (allSupportedParentCurrencies.find(item => item === token.currencyCode)) {
```

Comment:
This is the other place custom tokens can get into the system - from the settings. If both of these entry points are shut down, then there will not be any conflicting tokens, and the other locations do not need filters.